geometry_msgs/Point point
uint8 frequency